src/Esfa.Vacancy.Register.Domain/Repositories/IFrameworkCodeRepository.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Esfa.Vacancy.Register.Domain.Repositories
+{
+    public interface IFrameworkCodeRepository
+    {
+        Task<IEnumerable<string>> GetAsync();
```

Comment:
GetAsync but it can be a little more descriptive. I know it can be inferred from the class name.

Reply:
😨